lib/cli-engine/formatters/_manifest.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
Because this is just data, can we just create a JSON file here? I'd call it `formatter-data.json` and you'd still be able to `require()` it, but would otherwise be ignored.

Reply:
sgtm, i like this approach